agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java

```diff
@@ -349,6 +349,21 @@ private DiseaseDownloadRow getBaseDiseaseDownloadRow(DiseaseAnnotationDocument a
 			row.setSource(sourceProvider);
 		}
 */
+		if (annotation.getExperimentalConditionList() != null) {
+			row.setExperimentalCondition(annotation.getExperimentalConditionsAggregated());
+		} else if (annotation.getPrimaryAnnotations() != null) {
+			for(org.alliancegenome.curation_api.model.entities.DiseaseAnnotation da : annotation.getPrimaryAnnotations()) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'for' is not followed by whitespace.
<!-- __reviewdog__:ChA4MjdmZWQ5Yjc5ZjZjYTAxEgpjaGVja3N0eWxl -->